tmux.py

```diff
@@ -93,19 +93,19 @@ def execute(self):
             sublime.error_message('tmux: ' + str(exception))
 
 class OpenTmuxCommand(sublime_plugin.WindowCommand, TmuxCommand):
-    def run(self, path=None, split=None):
-        if not path:
-            path = self.resolve_file_path()
+    def run(self, paths=[], split=None):
+        dirs = [os.path.dirname(path) if not os.path.isdir(path) else path for path in paths]
```

Comment:
Note that the `dirs` argument was originally tried, and worked for sidebar items. However, with multiple projects open in separate windows, Sublime would report this argument incorrectly as if cached for the first focused project, despite `self.window.active_view().file_name()` resolving correctly. `paths` is the best working option, and this file system call is necessary.